plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1512,6 +1514,7 @@ modelement.villagertrade=Villager trade
 modelement.unknown.description=This mod element type no longer exists
 modelement.achievement.description=Advancement is a text displayed when user does something that is required to achieve.
 modelement.armor.description=Armor is an item that can be dressed up and gives you protection.
+modelement.bannerpattern.description=Banner pattern adds new patterns that can be displayed on banners and shields.
```

Comment:
I'd change the description. The keyword `adds` is not in any of other descriptions. You can find a better one.